pkg/vm/vm_core.go

```diff
@@ -22,17 +22,15 @@ type VirtualMachine struct {
 	RunContext     RunContext
 	CurrentStep    uint
 	Segments       memory.MemorySegmentManager
-	BuiltinRunners []builtins.BuiltinRunner
 	Trace          []TraceEntry
 	RelocatedTrace []RelocatedTraceEntry
 }
 
 func NewVirtualMachine() *VirtualMachine {
 	segments := memory.NewMemorySegmentManager()
-	builtin_runners := make([]builtins.BuiltinRunner, 0, 9) // There will be at most 9 builtins
 	trace := make([]TraceEntry, 0)
 	relocatedTrace := make([]RelocatedTraceEntry, 0)
-	return &VirtualMachine{Segments: segments, BuiltinRunners: builtin_runners, Trace: trace, RelocatedTrace: relocatedTrace}
+	return &VirtualMachine{Segments: segments, Trace: trace, RelocatedTrace: relocatedTrace}
```

Comment:
Why is this being removed? This was added [here](https://github.com/lambdaclass/cairo_vm.go/pull/29/files#diff-551d60bb3ab8d90961c72f009aeae898c3fb0378518ba252997e3fe434229104)

Reply:
it seems that came from a merge conflict